spec/visual/samples/rego

```diff
@@ -0,0 +1,27 @@
+package httpapi.authz
```

Comment:
Could you add some additional examples to this file to make sure the lexing is being done correctly? At the moment there are no numbers, no single-quoted strings, etc.

Reply:
Added numbers; single-quote strings are actually invalid in the language, so there's no valid lex for them. I think that covers all of the facets of the syntax; it's pretty simple!

Reply:
I might have used the wrong term here. In the `atoms` state, there's a rule matching single-quoted strings that generates `Str::Symbol` tokens. This is what I had in mind.

Reply:
Ah, I'd mentally glossed over that. I've just removed that rule, since it's not a valid lex to begin with.